src/wait-for-element.js

```diff
@@ -46,7 +46,9 @@ function waitForElement(
       onDone(lastError || new Error('Timed out in waitForElement.'), null)
     }
     timer = setTimeout(onTimeout, timeout)
-    observer = new MutationObserver(onMutation)
+    const window = container.ownerDocument.defaultView
+    const MutationObserverConstructor = window.MutationObserver || MutationObserver
```

Comment:
This code is not necessary as `@sheerun/mutationobserver-shim` already uses global MutationObserver if available

Reply:
Oh nice! Yeah, when you work on the merge conflicts, could you fix that?

Thanks for all the work your putting into this 👍